worker/src/main/java/gov/cms/ab2d/worker/adapter/bluebutton/PatientClaimsProcessorImpl.java

```diff
@@ -115,7 +115,7 @@ private void appendToFile(Path outputFile, ByteArrayOutputStream byteArrayOutput
         tryLock(lock);
 
         try {
-            log.info("Attempting to append to file", keyValue(FILE_LOG, outputFile.getFileName()));
+            log.info("Attempting to append to file", keyValue(FILE_LOG, outputFile.toFile().getName()));
             fileService.appendToFile(outputFile, byteArrayOutputStream);
```

Comment:
We found this logging statement was logging the path in addition to the filename while we were prepping for the demo during PI planning/roadmap session. I saw this while testing this change and fixed it.